benchmark/sirun/dogstatsd.js

```diff
@@ -0,0 +1,59 @@
+'use strict'
+
+const dgram = require('dgram')
+
+class DogStatsD {
+  constructor (options) {
+    options = options || {}
+
+    this._prefix = options.prefix || ''
+    this._tags = options.tags || []
+    this._buffer = ''
+    this._udp = this._socket('udp4')
+  }
+
+  gauge (stat, value, tags) {
+    this._add(stat, value, 'g', tags)
+  }
+
+  increment (stat, value, tags) {
```

Comment:
only gauge is currently supported by sirun.

Reply:
Removed `increment`.